mne_neuron/dipole.py

```diff
@@ -15,22 +17,44 @@ def _hammfilt(x, winsz):
     win /= sum(win)
     return convolve(x, win, 'same')
 
-
-def simulate_dipole(net):
+def simulate_dipole(fn):
     """Simulate a dipole given the experiment parameters.
 
     Parameters
     ----------
-    net : Network object
-        The Network object specifying how cells are
-        connected.
+    fn : string
+        The filename of the params file
 
     Returns
     -------
     dpl: instance of Dipole
         The dipole object
+    net : Network object
+        The Network object specifying how cells are
+        connected.
```

Comment:
I had it this way before but it leads to a less flexible API. Is this necessary for the parallel context to work? The way I see it, the network object specifies how cells are connected, so the user should have access to it and be able to add layers of neurons and specify connectivity structure.

Reply:
Part of the problem is starting up neuron before calling simulate_dipole. There's probably another way to do it.

Reply:
Reverting this pull request to the original method